### PR TITLE
Minor word chang AAA 244 historic literature

### PR DESCRIPTION
**Pull request by BartlettAstro**
changes "weed" to "reduce" to avoid suggesting that hte librarian doing the weeding thought the literature they were sending us was wor

### CODE REVIEW

File: _includes/_blogcontent/2024-06-04-HistoricLit.md [hunk 1]
```diff
@@ -12,7 +12,11 @@ Through the mid-20th century, many observatories had their own outlets for disse
...
  
+<<<<<<< AAS-244-HistoricLiterature
+NASA SciX continues to digitize legacy materials as we receive them. Librarians and researchers send us publications as they reduce their collections. Small astronomy organizations have asked us to become their repository. We estimate our historic astronomy literature collection is over 80% complete. We measure completeness by the fraction of papers that are referenced articles in the core astronomy and astrophysics journals for which we have entries.  
+=======
 NASA SciX continues to digitize legacy materials as we receive them. Librarians and researchers send us publications as they weed their collections. Small astronomy organizations have asked us to become their repository. We estimate our historic astronomy literature collection is over 80% complete. We measure completeness by the fraction of papers that are referenced articles in the core astronomy and astrophysics journals for which we have entries.  
+>>>>>>> master
```
Comment (mugdhapolimera) [inline _includes/_blogcontent/2024-06-04-HistoricLit.md:19]: I think something got funky when you merged some changes? this version has kept both the master and the branch version of this file. Also I cant see the other file changes that you made. Maybe something accidentally merged into master?

**Review by mugdhapolimera — CHANGES_REQUESTED** [review]

File: _includes/_blogcontent/2024-06-04-HistoricLit.md (hunk 1 above)
Comment (BartlettAstro) [reply _includes/_blogcontent/2024-06-04-HistoricLit.md]: I only had a single word change for this request. I'm not sure what happened, but it looks like something went wrong when I tried to resolve the conflict before you reviewed it.

**Comment by mugdhapolimera** [comment]
Hmm. That's weird. Maybe try making a new PR with the change?

On Jun 6, 2024 22:24, Jennifer Lynn Bartlett ***@***.***> wrote:

@BartlettAstro commented on this pull request.

________________________________

In _includes/_blogcontent/2024-06-04-HistoricLit.md<https://github.com/adsabs/adsabs.github.io/pull/465#discussion_r1630545398>:

> +<<<<<<< AAS-244-HistoricLiterature
+NASA SciX continues to digitize legacy materials as we receive them. Librarians and researchers send us publications as they reduce their collections. Small astronomy organizations have asked us to become their repository. We estimate our historic astronomy literature collection is over 80% complete. We measure completeness by the fraction of papers that are referenced articles in the core astronomy and astrophysics journals for which we have entries.
+=======
 NASA SciX continues to digitize legacy materials as we receive them. Librarians and researchers send us publications as they weed their collections. Small astronomy organizations have asked us to become their repository. We estimate our historic astronomy literature collection is over 80% complete. We measure completeness by the fraction of papers that are referenced articles in the core astronomy and astrophysics journals for which we have entries.
+>>>>>>> master


I only had a single word change for this request. I'm not sure what happened, but it looks like something went wrong when I tried to resolve the conflict before you reviewed it.

**Review by mugdhapolimera — APPROVED** [review]
looks good. feel free to merge.

**Pull request merged**